libraries/datahandler/src/main/java/org/eclipse/sw360/datahandler/permissions/PermissionUtils.java

```diff
@@ -174,4 +174,44 @@ public static <T> DocumentPermissions<T> makePermission(T document, User user) {
         }
     }
 
+    public static Set<String> editableParmsSet = new HashSet<>();
```

Comment:
Please rename the variable to be more specific. Something like `closedProjectEditableParams`.

It will be best if it can be made `final` as well, since it is `public`. Then it can be renamed with proper capital casing (example `CLOSED_PROJECT_EDITABLE_PARAMS`).